### DISCUSSION TITLE
`rootRedirect` available on v8?

### DISCUSSION THREAD

**Discussion by quentint** [nuxt-modules/i18n#1553] [Q&A]
Hey,  
can you confirm the `rootRedirect` option is not available on v8 (`^8.0.0-alpha.3`)?

If so, is this something I should wait for, or is there another way for my root (`pages/index.vue` > `/`) to redirect to the default locale path (`/en`)?

Thanks for your help!

**Comment by ineshbose** [comment]
There is an equivalent to this option in v8 depending on your routing `strategy`. You can read more here https://v8.i18n.nuxtjs.org/guide/routing-strategies and `prefix` will cause redirects: https://v8.i18n.nuxtjs.org/guide/routing-strategies#prefix.

However there is a bug at the moment (that should be fixed soon) causing errors in production during this redirect: https://github.com/nuxt-community/i18n-module/issues/1611
